active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,618 @@
+### Remarks
+There is one Compositor per thread, that means that all the Windows in the same thread share the same Compositor.
+
+
+
+## WindowEventArgs class
+Namespace: Microsoft.UI.Xaml
+
+Contains the set of arguments returned to an app after a window input or behavior event.
+
+### Properties
+
+- **WindowEventArgs.Handled**: Gets or sets whether the event was handled. **true** if the event has been handled by the appropriate delegate; **false** if it has not.
+    ```CS
+    public bool Handled { get; set; }
+    ```
+
+
+## WindowActivatedEventArgs class
+Namespace: Microsoft.UI.Xaml
+
+Contains the windows activation state information returned by the Window.Activated event.
+
+```CS
+public class WindowActivatedEventArgs
+```
+
+### Properties
+- **WindowActivatedEventArgs.Handled**: Gets or sets whether the window activation changed event was handled. **true** if the event has been handled by the appropriate delegate; **false** if it has not.
+    ```CS
+    public bool Handled { get; set; }
+    ```
+
+
+- **WindowActivatedEventArgs.WindowActivationState**: Gets the activation state of the window at the time the event was raised.
+    ```CS
+    public CoreWindowActivationState WindowActivationState { get; }
+    ```
+
+## VisibilityChangedEventArgs class 
+Namespace: Microsoft.UI.Xaml
+
+
+Contains the arguments returned by the event fired when a Window instance's visibility changes.
+```CS
+public class VisibilityChangedEventArgs
+```
+
+### Properties:
+- VisibilityChangedEventArgs.Handled: Gets or sets whether the window activation changed event was handled. **true** if the event has been handled by the appropriate delegate; **false** if it has not.
+    ```CS
+    public bool Handled { get; set; };
+    ```
+
+- VisibilityChangedEventArgs.Visible: Gets whether the window is visible or not. **true** if the window is visible; otherwise is **false**.
+    ```CS
+    public bool Visible { get; } 
+    ```
+
+## WindowSizeChangedEventArgs class
+Namespace: Microsoft.UI.Xaml
+
+Contains the argument returned by a window size change event.
+
+### Properties
+
+- WindowSizeChangedEventArgs.Handled: Gets or sets whether the  event was handled. **true** if the event has been handled by the appropriate delegate; **false** if it has not.
+    ```CS
+    public bool Handled { get; set; };
+    ```
+
+- WindowSizeChangedEventArgs.Visible: Gets the new size of the window in units of effective (view) pixels.
+    ```CS
+    public Size Visible { get; } 
+    ```
+
+
+## Application class
+Namespace: Microsoft.UI.Xaml
+
+XAML Application exposes special support for managing the Windows in an application. 
+
+
+
+```CS
+public class Application:IClosable
+```
+```XML
+<Application />
+ ```
+
+> **NOTE**: It is not a goal of this doc to review all the members of this class. Just that ones that requires changes in Win32. You can find more inforation about Application class [here](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.application).
```

Comment:
I wish this was on top, because I just did that :)

Reply:
We change the scope of this spec, and only cover the updates, so this comment is outdated. Resolving it.